list-sources.js

```diff
@@ -0,0 +1,13 @@
+const {readdirSync} = require('fs');
```

Comment:
This doesn't work with older node versions (see https://ci.appveyor.com/project/parro-it/libui-node/build/1.0.279/job/p4lskfm3yfr5b2xt#L568).

More conservative syntax:
```js
var readdirSync = require('fs').readdirSync;
```

Reply:
Yes thank you, I saw it, just pushed change.